src/cascadia/TerminalApp/pch.h

```diff
@@ -7,49 +7,6 @@
 
 #pragma once
 
-#define WIN32_LEAN_AND_MEAN
-
-#include <LibraryIncludes.h>
-// This is inexplicable, but for whatever reason, cppwinrt conflicts with the
-//      SDK definition of this function, so the only fix is to undef it.
-// from WinBase.h
-// Windows::UI::Xaml::Media::Animation::IStoryboard::GetCurrentTime
-#ifdef GetCurrentTime
-#undef GetCurrentTime
-#endif
-
-#include <unknwn.h>
-
-#include <hstring.h>
-
-#include <winrt/coroutine.h>
-
-#include <winrt/Windows.ApplicationModel.Resources.h>
-#include <winrt/Windows.Foundation.h>
-#include <winrt/Windows.Foundation.Collections.h>
-#include <winrt/windows.ui.core.h>
-#include <winrt/Windows.ui.input.h>
-#include <winrt/Windows.UI.Text.h>
-#include <winrt/Windows.UI.Xaml.Controls.h>
-#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
-#include <winrt/Windows.ui.xaml.media.h>
-#include <winrt/Windows.ui.xaml.input.h>
-#include <winrt/Windows.UI.Xaml.Hosting.h>
-#include <winrt/Microsoft.Toolkit.Win32.UI.XamlHost.h>
-
-#include <windows.ui.xaml.media.dxinterop.h>
-
-#include <winrt/Windows.System.h>
-
-// Including TraceLogging essentials for the binary
-#include <TraceLoggingProvider.h>
-#include <winmeta.h>
-TRACELOGGING_DECLARE_PROVIDER(g_hTerminalWin32Provider);
-#include <telemetry\ProjectTelemetry.h>
-#include <TraceLoggingActivity.h>
-
-// JsonCpp
-#include <json.h>
-
-#include <shellapi.h>
-#include <filesystem>
+// This file can be empty - the pch.h in TerminalApp/lib does the heavy lifting

```

Comment:
If this file is just gonna be empty, could/should we remove it?

Reply:
Unfortunately, MsBuild needs the pch to exist for any cppwinrt projects. It's going to autogenerate other files that depend on including it, even if it's empty #bydesign